galsim/lensing_ps.py

```diff
@@ -1617,7 +1617,7 @@ def __call__(self, gd):
         if self.amplitude_E is not None:
             r1 = galsim.utilities.rand_arr(self.amplitude_E.shape, gd)
             r2 = galsim.utilities.rand_arr(self.amplitude_E.shape, gd)
-            E_k = np.empty((self.ny,self.nx)).astype(type(1.+1.j))
+            E_k = np.empty((self.ny,self.nx), dtype=type(1.+1.j))
```

Comment:
Sounds good.

Also... I was playing around and noticed that ```empty_like``` seems to be faster than ```empty``` but ```zeros_like``` seems slower than ```zeros```, which kind of confuses me.  Perhaps my test cases are too contrived, or this is system-dependent; is there a reason this would make sense?  If it's right, then that might motivate changing around some of our ```empty``` vs. ```empty_like```, but have you ever seen this?